Panels/tests/Panels-specs.js

```diff
@@ -1,13 +1,73 @@
 import '@testing-library/jest-dom';
-import {render, screen} from '@testing-library/react';
+import {fireEvent, render, screen} from '@testing-library/react';
 
 import Button from '../../Button';
 import Header from '../../Header';
 import Panel from '../Panel';
 import Panels, {PanelsBase} from '../Panels';
 
+const keyDown = (keyCode) => (elm) => fireEvent.keyDown(elm, {keyCode});
+const keyUp = (keyCode) => (elm) => fireEvent.keyUp(elm, {keyCode});
+
+const escapeKeyUp = keyUp(27);
+const escapeKeyDown = keyDown(27);
+
 describe('Panels Specs', () => {
-	test('should set {autoFocus} on child to "default-element" on first render', () => {
+	test('should display the correct panel based on \'index\' prop', () => {
+		let firstButton, secondButton;
+		const {rerender} = render(
+			<Panels index={0} noAnimation>
+				<Panel>
+					<Button>One</Button>
+				</Panel>
+				<Panel>
+					<Button>Two</Button>
+				</Panel>
+			</Panels>
+		);
+
+		firstButton = screen.getByText('One');
+		secondButton = screen.queryByText('Two');
+
+		expect(firstButton).toBeInTheDocument();
+		expect(secondButton).not.toBeInTheDocument();
+
+		rerender(
+			<Panels index={1} noAnimation>
+				<Panel>
+					<Button>One</Button>
+				</Panel>
+				<Panel>
+					<Button>Two</Button>
+				</Panel>
+			</Panels>
+		);
+
+		firstButton = screen.queryByText('One');
+		secondButton = screen.getByText('Two');
+
+		expect(firstButton).not.toBeInTheDocument();
+		expect(secondButton).toBeInTheDocument();
+
+		rerender(
+			<Panels index={0} noAnimation>
+				<Panel>
+					<Button>One</Button>
+				</Panel>
+				<Panel>
+					<Button>Two</Button>
+				</Panel>
+			</Panels>
+		);
+
+		firstButton = screen.getByText('One');
+		secondButton = screen.queryByText('Two');
```

Comment:
```suggestion
		firstButton = screen.queryByText('One');
		secondButton = screen.queryByText('Two');
```

I think it's good to unify the functions for the fairness of the test. I'll fix it.